assets/dev/js/frontend/utils/anchor-scroll-margin.js

```diff
@@ -0,0 +1,160 @@
+module.exports = elementorModules.ViewModule.extend( {
```

Comment:
```suggestion
export default class extends elementorModules.ViewModule
```
Can you make this an ES6 module?